spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/query/JpqlComplianceTests.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 the original author or authors.
```

Comment:
Nit: 2024

Reply:
3 weeks ago this had been valid ;) - I'll change after rebase.